ts/components/session/SessionPasswordPrompt.tsx

```diff
@@ -31,9 +31,17 @@ export class SessionPasswordPrompt extends React.PureComponent<{}, State> {
     window.addEventListener('keyup', this.onKeyUp);
   }
 
+  public componentDidMount() {
+    setTimeout(() => $('#password-prompt-input').focus(), 100);
+  }
+
+  public componentWillUnmount() {
+    window.removeEventListener('keyup', this.onKeyUp);
```

Comment:
to remove if handler is set in in the input

Reply:
Also, the 
`     window.addEventListener('keyup', this.onKeyUp);`
should probably not be in the constructor, rather in the `componentDidMount` method instead